### PR TITLE
Improve code tests and comments

### PR DESCRIPTION
**Pull request by lingshanng**

### CODE REVIEW

File: src/main/java/seedu/address/model/lesson/CalendarEntryList.java [hunk 1]
```diff
@@ -381,7 +381,7 @@ public void updateUpcomingLessons() {
      *
      * @param owner The person with the lesson.
      * @param lesson The lesson associated with this entry.
-     * @return The calendar entry that also contains this lesson.
+     * @return Then calendar entry of the lesson.
```
Comment (eeliana) [inline src/main/java/seedu/address/model/lesson/CalendarEntryList.java:384]: What is then calendar?

Comment (Chesterwongz) [inline src/main/java/seedu/address/model/lesson/CalendarEntryList.java:384]:
```suggestion
     * @return The calendar entry of the lesson.
```

File: src/main/java/seedu/address/model/lesson/CalendarEntryList.java [hunk 2]
```diff
@@ -394,7 +394,7 @@ public void updateUpcomingLessons() {
      * @param owner The person with the lesson.
      * @param lesson The lesson associated with this entry.
      * @param entryInterval The interval of this entry.
-     * @return The calendar entry that also contains this lesson.
+     * @return Then calendar entry of the lesson.
```
Comment (Chesterwongz) [inline src/main/java/seedu/address/model/lesson/CalendarEntryList.java:397]:
```suggestion
     * @return The calendar entry of the lesson.
```

**Review by Chesterwongz — APPROVED** [review]
lgtm

**Review by eeliana — APPROVED** [review]

**Pull request merged**